### ISSUE TITLE
Alternate Collection/Deck Display

### ISSUE THREAD

**Issue by mkaatman** [monovertex/ygorganizer#24]
Future: Ability to toggle the display from rows to a more visual display such as a grid of card photos.

**Comment by monovertex** [comment]
For the deck builder itself, it's already like that, although that feature is not public yet.

So this would be for the card lists (collection, browse, required cards for decks).

Do you think we could display any additional data besides images? I think that would become too heavy, we should just stick to a simple grid.

**Comment by mkaatman** [comment]
I agree, seems like it could get a bit too busy with more than the image. Perhaps you can still click the card to get the data view (similar to what we have today) of that card in some form.

**Comment by monovertex** [comment]
Well, I would definitely keep the current card viewer and just the table display would be changed.

Ok, only images it is. Should this view disable sorting? How would we handle that, considering the column names no longer have a place in this view?

**Comment by mkaatman** [comment]
Maybe a couple of subtle buttons that allow you to sort on the header by type, name, level or something like that?

It might be neat to have a deck profile chart(s) the way hearthstone does with mana cost. I'm not sure it's as relevant here or what it would be profiling exactly. Something to think about.
http://img.youtube.com/vi/paH2zVOGHWI/maxresdefault.jpg

**Comment by monovertex** [comment]
I am not really sure about what it would profile either. We don't have a single important resource like Hearthstone does.

We'll see, but I'm not considering the curve analysis important for the moment.

**Comment by mkaatman** [comment]
I was thinking along the lines of showing how many of each attribute or level. Certainly not an important feature.
